### ISSUE TITLE
GUI 5.0.0 - character filter issue at step 4

### ISSUE THREAD

**Issue by MeijeSibbel** [storj/storjshare-gui#515]
### Versions

Replace the values below with your own:

- **GUI Version:** v1.0.3
- **Operating System:** Ubuntu 14.04

### Expected Behavior

Please describe the program's expected behavior.

```
I am not able to enter characters in the port filed, i should only be able to enter numerical values. 
```

### Actual Behavior

Please describe the program's actual behavior. Please include any stack traces
or log output in the back ticks below.

```
i am able to enter the character `e` into the port field. 
```

### Steps to Reproduce

Please include the steps the reproduce the issue, numbered below. Include as
much detail as possible.

1. ...
2. ...
3. ...

### Screenshots (Optional)

![issue2](https://cloud.githubusercontent.com/assets/11735738/24636331/8218e8c2-18d1-11e7-98b5-8cb959af58cb.png)

**Comment by MeijeSibbel** [comment]
I am also able to enter other special symbols such as `.` , `+`, `-`

**Comment by littleskunk** [comment]
Should be fixed with #510

**Comment by RichardLitt** [comment]
👋 Hey! Thanks for this contribution. Apologies for the delay in responding!

We've decided to rearchitect Storj, so that we can scale better. You can read more about this decision [here](https://storj.io/blog/2018/01/getting-from-petabytes-to-exabytes-the-road-ahead/). This means that we are entirely focused on v3 at the moment, in the [storj/storj](https://github.com/storj/storj) repository. Our white paper for v3 is coming **very, very** soon - follow along on [the blog](https://storj.io/blog) and in our [Rocketchat](https://community.storj.io/home).

As this repository is part of the v2 network, we're no longer maintaining this repository. I am going to close this for now. If you have any questions, I encourage you to jump on Rocketchat and ask them there. Thanks!
